extensions/amp-story/amp-story.md

```diff
@@ -110,6 +110,66 @@ The new attributes:
 | `poster-square-src` | The story poster in square format (1x1 aspect ratio). |
 | `poster-landscape-src` | The story poster in landscape format (4x3 aspect ratio). |
 
+#### `publisher-logo-src` guidelines
+
+The following guidelines apply to the image for the publisher logo:
+
+- The file must be a raster file, such as `.jpg`, `.png`, or `.gif`.  Don't use vector files, such as `.svg` or `.eps`.
+- Don't use animation.
```

Comment:
Can you clarify what you mean?  If it's a gif, then it might be animated, right? So what do you mean by "don't use animation".  Also, I would avoid negatives and use "Avoid" vs "Don't .."